### DISCUSSION TITLE
Bootloader Update and Redundancy

### DISCUSSION THREAD

**Discussion by dat-tran-matternet** [OE4T/meta-tegra#631] [Q&A] [answered]
Does meta-tegra support bootloader update and redundancy as shown here:
https://docs.nvidia.com/jetson/l4t/index.html#page/Tegra%20Linux%20Driver%20Package%20Development%20Guide/bootloader_update.html#

Thanks

**Comment by ichergui** [comment, marked as answer]
Hey @dat-tran-matternet 
A quick response is Yes.
Could you please give us more details about what you want to do ?

**Reply by dwalkes** [reply]
The mender implementation in https://github.com/OE4T/tegra-demo-distro/ is a good place to find an example implementation.

**Reply by dat-tran-matternet** [reply]
We want to be able to deploy OTA updates that can update all the partitions (MB1, MB2, CBOOT, UBOOT, KERNEL, etc) and be able to roll back if any fails during that attempt. Also in the future will need secure boot but we are still learning the platform so we will go slowly. Thanks for helping
